src/framework/auth/services/token/token.ts

```diff
@@ -17,11 +19,18 @@ export interface NbAuthRefreshableToken {
 
 export interface NbAuthTokenClass {
   NAME: string;
-  new (raw: any): NbAuthToken;
+  new (raw: any, ownerStrategyName: string): NbAuthToken;
 }
 
-export function nbAuthCreateToken(tokenClass: NbAuthTokenClass, token: any) {
-  return new tokenClass(token);
+// All types of token are not refreshables
+export function isNbAuthRefreshableToken(token: any): token is NbAuthRefreshableToken {
```

Comment:
I guess this is not a part of this PR, isn't it?

Reply:
you're right. Part of the next next next :)